src/js/rtc_session.js

```diff
@@ -480,26 +480,38 @@ export default class RtcSession {
     get callId() {
         return this._callId;
     }
+    /**
+     * getMediaStream returns the client provided stream.
+     * Rather than getting a stream by calling getUserMedia (which gets a stream from local device such as camera),
+     * user could also provide the stream to the RtcSession directly to connect to the other end.
+     * This method returns the client provided stream.
+     */
     get mediaStream() {
-        return this._userAudioStream;
+        return this._localStream;
     }
+    /**
+     * getLocalVideoStream returns user's local video stream, which may come from local device such as camera,
+     * or from the client provided stream.
+     */
     get localVideoStream() {
-        return this._streamToBeClosed;
+        return this._streamToBeClosed || this._localStream;
```

Comment:
should only use _localStream. Depends on your use case, we should either:
1. deprecate this method, or
2. filter out non-video tracks and wrap into a new stream